### DISCUSSION TITLE
How to link to pdf's in text

### DISCUSSION THREAD

**Discussion by dherrera1911** [academicpages/academicpages.github.io#2307] [Q&A]
Is it possible to link to files in the 'files' directory from the text that the user has to input? For example, I'd like to have a supporting pdf document in 'files', that I link to in the description of one of my papers.

I guess it should not be difficult to do this, but it's confusing to me how to, because if I understand correctly, it's markdown converted to html with python scripts, then rendered with jekyll, which is too much for me to figure out.

**Comment by rjzupkoii** [comment]
Can you clarify what you are going for? As I understand it you would like to be able to link to files on your website from a PDF - that should work just fine once the website as been deployed as long as you know what the URL is to link to in the PDF.

**Reply by dherrera1911** [reply]
Thanks for responding. I figured out how to do what I wanted, and posted as an answer below.

**Comment by dherrera1911** [comment]
I figured out how to link to pdfs kept in the /files directory, it was fairly easy.  E.g. If I want to have a PDF with supporting information for a paper in the /files direectory, and to link to this in the text for that publication (the white text at the end of the .md file), you just  do [text about pdf](../files/myfile.pdf).
